tests/Formatting/Formatters/FullyQualifiedFacadesTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Tests\Formatting\Formatters;
+
+use PHPUnit\Framework\TestCase;
+use Tighten\TLint\Formatters\FullyQualifiedFacades;
+use Tighten\TLint\TFormat;
+
+class FullyQualifiedFacadesTest extends TestCase
+{
+    /** @test */
+    public function fixes_facade_aliases()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use DB;
+use Storage;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+namespace Test;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_facade_aliases_when_file_not_namespaced()
```

Comment:
Should we only fix on namespaced files?

Reply:
I *think* I'm ok with us using the alias in non-namespaced files, but I don't really have a super strong opinion?

Reply:
Just asked everyone in Tighten to vote, so we'll know soon.

Reply:
Looks like the vote was for consistency:

```php
<?php

use Illuminate\Support\Facades\Cache;

$abc = Cache::get('abc');
```

```php
<?php

namespace App\Utilities;

use Illuminate\Support\Facades\Cache;

class SuperCacher
{
    public function doThing()
    {
        return Cache::get('abc');
    }
}
```

Reply:
Yep, seems so! Honestly that's not my preference but I'm gonna go with the team :)